src/components/dropdownSearch/DropdownSearchConnected.tsx

```diff
@@ -0,0 +1,52 @@
+import { IReactVaporState, IReduxActionsPayload } from '../../ReactVapor';
+import { DropdownSearch, IDropdownOption, IDropdownSearchOwnProps, IDropdownSearchProps, IDropdownSearchStateProps } from './DropdownSearch';
+import { IReduxAction, ReduxUtils } from '../../utils/ReduxUtils';
+import { connect } from 'react-redux';
+import * as _ from 'underscore';
+import {
+  addDropdownSearch,
+  applyFilterDropdownSearch,
+  removeDropdownSearch,
+  selectOptionDropdownSearch,
+  toggleDropdownSearch,
+  updateActiveOptionDropdownSearch,
+} from './DropdownSearchActions';
+import { defaultSelectedOption, IDropdownSearchState } from './DropdownSearchReducers';
+
+const mapStateToProps = (state: IReactVaporState, ownProps: IDropdownSearchProps): IDropdownSearchStateProps => {
+  const dropdownSearch: IDropdownSearchState = _.findWhere(state.dropdownSearch, { id: ownProps.id });
+  const selectedOption = ownProps.defaultSelectedOption || defaultSelectedOption;
+  if (dropdownSearch) {
+    return {
+      isOpened: dropdownSearch.isOpened || false,
+      options: dropdownSearch.options || [],
+      selectedOption: dropdownSearch.selectedOption || selectedOption,
+      filterText: dropdownSearch.filterText || '',
+      activeOption: dropdownSearch.activeOption,
+      setFocusOnDropdownButton: dropdownSearch.setFocusOnDropdownButton,
+    };
+  }
+
+  return {
+    isOpened: false,
```

Comment:
You could forgo this return and defaults in the other return if you add these values as default props.